File: UCLAscript/test_DB.py
#!/usr/bin/env python
from UCLAscript import *
print "UCLAscript loaded OK!"
params=DB_Parameters()
params.alwaysCheckConsistency=1
db=DB("./alt0.aux",params)
netlist=db.getNetlist()
netlist.thisown=0
print "Testcase:",db.getAuxName()
print "Total CCs ",netlist.getNumCCs()
print "Total SCs ",netlist.getNumSCs()
print "Total AllPads ",netlist.getNumAllPads() 
netlist.printNetStats()

File: UCLAscript/test_DB.pl
#!/usr/bin/env perl 
BEGIN {unshift @INC,(".","blib/arch");}
use UCLAscript;
package UCLAscript;
$params=new DB_Parameters();
$params->{alwaysCheckConsistency}=1;
$db=new DB("./alt0.aux",$params);
$netlist=$db->getNetlist();
print "Testcase: ",$db->getAuxName(),"\n";
print "Total CCs ",$netlist->getNumCCs(),"\n";
print "Total SCs ",$netlist->getNumSCs(),"\n";
print "Total AllPads ",$netlist->getNumAllPads(),"\n"; 
$netlist->printNetStats();

